src/combo.ts

```diff
@@ -25,6 +29,58 @@ type BeachTile = { minute: number; row: number; column: number };
 // Unfortunately, this can mean some bad things could happen were our data file to get messed up
 const rareTiles: BeachTile[] = JSON.parse(fileToBuffer("raretiles.json"));
 
+// A list of all the common items found in infreqeunt twinkles to detect when we actually combed an infrequent tile instead of a rare
+const infrequentItemNames: string[] = [
+  "sand dollar",
+  "dull fish scale",
+  "rough fish scale",
+  "lucky rabbitfish fin",
+  "spearfish fishing spear",
+  "Waders",
+  "pristine fish scale",
+  "piece of coral",
+].map((s) => s.toLowerCase());
```

Comment:
Either do a $items and map it to `.name.toLowerCase()` or just make the strings lowercase to begin with